Packs/TOPdesk/Layouts/layoutscontainer-TOPdesk_incident.json

```diff
@@ -1217,104 +1217,8 @@
       }
     ]
   },
-  "quickView": {
-    "sections": [
-      {
-        "description": "",
-        "fields": [
-          {
-            "fieldId": "incident_type",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_severity",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_owner",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_dbotstatus",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_sourcebrand",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_sourceinstance",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_playbookid",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_phase",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_roles",
-            "isVisible": true
-          }
-        ],
-        "isVisible": true,
-        "name": "Basic Information",
-        "query": null,
-        "queryType": "",
-        "readOnly": false,
-        "type": ""
-      },
-      {
-        "description": "",
-        "fields": [
-          {
-            "fieldId": "incident_occurred",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_dbotcreated",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_dbotduedate",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_dbotmodified",
-            "isVisible": true
-          },
-          {
-            "fieldId": "incident_dbottotaltime",
-            "isVisible": true
-          }
-        ],
-        "isVisible": true,
-        "name": "Timeline Information",
-        "query": null,
-        "queryType": "",
-        "readOnly": false,
-        "type": ""
-      },
-      {
-        "description": "",
-        "fields": [
-          {
-            "fieldId": "incident_labels",
-            "isVisible": true
-          }
-        ],
-        "isVisible": true,
-        "name": "Labels",
-        "query": null,
-        "queryType": "",
-        "readOnly": true,
-        "type": "labels"
-      }
-    ]
-  },
```

Comment:
by design?

Reply:
this json has 2 identical quickview, hence not a valid json. Not sure how it passed the validations.